admin/bin/qserv-admin.py

```diff
@@ -154,7 +186,7 @@ def receiveCommands(self):
         cmd = ''
         prompt = self._prompt
         while True:
-            line = raw_input(prompt).decode("utf-8").strip()
+            line = raw_input(prompt).strip()
```

Comment:
I'll open a ticket to remind us to do it later. I think "right" approach to unicode is a to always handle strings in unicode internally and only encode them to pass to APIs that accept bytes. Question is should CssAccess accept bytes only or should it be better accept unicode.